src/pages/Projects/components/RoleModal/RoleModal.module.css

```diff
@@ -1,50 +1,47 @@
-.assignmentSelect__control {
-  padding: 20px;
-}
-
-.assignmentSelect div[class$="control"] svg {
+:global(.assignmentSelect__control) svg {
```

Comment:
You should add a class that follows the normal randomized rules that all of these fall into, so that these overrides _only_ after the uses inside RoleModal, and don't bleed out into other places on the site. This is often called `.root` or similar.

Reply:
added it and changed to scss to make it simpler